test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -675,8 +674,7 @@ def test_cgroup_cpuset_and_memory(self):
             with open(tmp_file, 'r') as fd:
                 tmp_out = fd.read().splitlines()
         except:
-            self.logger.info("Job did not produce expected output")
-            self.assertTrue(False)
+            self.assertTrue(False, "Job did not produce expected output")
```

Comment:
I took a closer look at fail and it doesn't do what we want.  It will fail the test but not include the failure in the failure count at the bottom.  Kind of a useless function if you ask me.  It's really skipping the test, not failing it.